pkg/operator/configobservation/serviceca/observeserviceca.go

```diff
@@ -0,0 +1,62 @@
+package serviceca
+
+import (
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+const (
+	serviceCAConfigMapName = "service-ca"
+	serviceCABundleKey     = "cabundle.crt"
```

Comment:
Update the other side to produce `ca-bundle.crt` and update here.  `must-gather` also understands `ca-bundle.crt`

Reply:
0222f146 - should start passing after https://github.com/openshift/service-ca-operator/pull/27